common/common.cpp

```diff
@@ -950,7 +950,7 @@ void gpt_print_usage(int /*argc*/, char ** argv, const gpt_params & params) {
     printf("  --override-kv KEY=TYPE:VALUE\n");
     printf("                        advanced option to override model metadata by key. may be specified multiple times.\n");
     printf("                        types: int, float, bool. example: --override-kv tokenizer.ggml.add_bos_token=bool:false\n");
-    printf("  -stc N --print-token-count N\n");
+    printf("  -stc N, --print-token-count N\n");
```

Comment:
This is something that I missed yesterday:

```suggestion
    printf("  -ptc N, --print-token-count N\n");
```